src/hal/components/toggle2nist.comp

```diff
@@ -4,61 +4,71 @@ description
 """
 Toggle2nist can be used with a momentary push button 
 to control a device that has separate on and off inputs
-and an is-on output. 
+and an is-on output.
+A debounce delay in cycles can be set. (default = 2)
 
 \[bu] On a rising edge on pin \\fIin\\fR when \\fIis-on\\fR is low: It sets \\fIon\\fR until \\fIis-on\\fR becomes high.
 
 \[bu] On a rising edge on pin \\fIin\\fR when \\fIis-on\\fR is high: It sets \\fIoff\\fR until \\fIis-on\\fR becomes low.
 
 
-       ┐     ┌─────xxxxxxxxxxxx┐           ┌─────xxxxxxxxxxxx┐
+       ┐     ┌─────xxxxxxxxxxxx┐           ┌─────xxxxxxxxxxxx┐
```

Comment:
Just for curiosity - what has changed here?

Reply:
I have asked myself the same as I have not changed anything in those lines. One thing I noticed is that the current man page did not display this diagram correctly before but it does seem to now (at least on my machine). I noticed there is a something about this in the commit history:
https://github.com/LinuxCNC/linuxcnc/pull/3007/files 

So not sure what this means, @petterreinholdtsen would probably know but if saving the file in a text editor breaks the diagram then it may be better to  just remove it.

Reply:
I managed to fix this by copying the spacing from the 'toggle' component.